payment_success.php

```diff
@@ -37,8 +35,8 @@
 
 			</div>
 			<ul class="nav navbar-nav">
-				<li><a href="#"><span class="glyphicon glyphicon-home"></span> Nyumbani</a></li>
-				<li><a href="#"><span class="glyphicon glyphicon-tags"></span> Vifaa</a></li>
+				<li><a href="profile.php"><span class="glyphicon glyphicon-home"></span> Nyumbani</a></li>
+				<li><a href="profile.php"><span class="glyphicon glyphicon-tags"></span> Vifaa</a></li>
```

Comment:
Since "Vifaa" and "Nyumbani" perform the same work, I would suggest we should just let only one remain. Lets keep "Nyumbani" and delete the line of code for "Vifaa"

Reply:
already corrected madam